ortools/math_opt/solver_tests/ip_model_solve_parameters_tests.cc

```diff
@@ -151,6 +151,17 @@ TEST_P(MipSolutionHintTest, TwoHintTest) {
   if (!TwoHintParams().has_value()) {
     GTEST_SKIP() << "Multiple hints not supported. Ignoring this test.";
   }
+  if (GetParam().solver_type == SolverType::kXpress) {
```

Comment:
I think that you should rather create specifc options and flags to allow disabling these 3 tests (to avoid circular dependency beween xpress code & test code). Like what you did in logical_constraint_tests.